Source/WebCore/platform/graphics/FontSizeAdjust.h

```diff
@@ -61,8 +61,10 @@ struct FontSizeAdjust {
         ChWidth,
         IcWidth,
         IcHeight
-    } metric;
-    Markable<float, FloatMarkableTraits> value;
+    };
+    
+    Metric metric { };
+    Markable<float, FloatMarkableTraits> value { };
```

Comment:
Isn't that exactly the same with 4 more characters ? (aka calling the default constructor) 

`Foo foo` and `Foo foo { }` ?

Reply:
I thought this too but seems like it is not. At least on Mac. Maybe because of `clang`.

Reply:
Can we write a test case which demonstrate it ?

Reply:
I've added an API test.

Reply:
I've updated the test. It should now better illustrate the problem.